docs/tutorial/using-checkbox/checkbox-commands.rst

```diff
@@ -0,0 +1,296 @@
+.. _base_tutorial_commands:
+
+============================
+A few more Checkbox commands
+============================
+
+Checkbox comes with a bunch of commands. To see them, run the following in
+a terminal:
+
+.. code-block:: none
+
+    checkbox.checkbox-cli --help
+
+    usage: checkbox-cli [-h] [-v] [--debug] [--clear-cache] [--clear-old-sessions] [--version]
+                        {check-config,launcher,list,run,startprovider,submit,show,list-bootstrapped,merge-reports,merge-submissions,tp-export,service,remote}
+
+    positional arguments:
+      {check-config,launcher,list,run,startprovider,submit,show,list-bootstrapped,merge-reports,merge-submissions,tp-export,service,remote}
+                            subcommand to run
```

Comment:
Maybe we can sacrifice accuracy for readability here and reformat the output a little bit to at least go to a newline on 80ish chars, I dont like scrolling the text box

Reply:
Fixed in [20d9258](https://github.com/canonical/checkbox/pull/635/commits/20d92587d4c1323c44666a647f24b991ccabe62d)